### ISSUE TITLE
should not publish tests folder

### ISSUE THREAD

**Issue by HsuTing** [cameronhunter/prettier-package-json#11]
I try to use `flow` and it throw this:

```js
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/prettier-package-json/tests/__fixtures__/invalid.json:2:10

Unexpected identifier

     1│ {
     2│     this is: invalid JSON
     3│ }
     4│
```

I can ignore this, but I think this folder should not be published.

**Comment by cameronhunter** [comment]
Agreed. I'll fix and publish tomorrow.

**Comment by HsuTing** [comment]
Thank you 😸

**Comment by cameronhunter** [comment]
Fixed and published as [v2.0.1](https://github.com/cameronhunter/prettier-package-json/releases/tag/v2.0.1). Thanks for reporting.
